container/docker/docker.go

```diff
@@ -132,6 +133,10 @@ func Version() ([]int, error) {
 	return parseDockerVersion(VersionString())
 }
 
+func APIVersion() ([]int, error) {
+	return parseDockerAPIVersion(APIVersionString())
+}
+
 func VersionString() string {
```

Comment:
Wondering if there should be a TODO: about removing the VersionSting() in a future release.